app/src/main/java/com/github/sdpteam15/polyevents/view/fragments/admin/EventEditDifferenceFragment.kt

```diff
@@ -0,0 +1,115 @@
+package com.github.sdpteam15.polyevents.view.fragments.admin
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import com.github.sdpteam15.polyevents.R
+import com.github.sdpteam15.polyevents.helper.HelperFunctions
+import com.github.sdpteam15.polyevents.model.entity.Event
+
+
+/**
+ * Fragment that will show the difference between an event edit request and the original event
+ * @param modifyEvent The proposed modification
+ * @param creation True if it is an event creation, false otherwise
+ * @param origEvents The original event or null if the request is an event creation
+ */
+class EventEditDifferenceFragment(
+    private val modifyEvent: Event,
+    private val creation: Boolean,
+    private val origEvents: Event?
+) : DialogFragment(R.layout.fragment_event_edit_difference) {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_event_edit_difference, container, false)
+        setupFieldContents(view)
+        setupFieldVisibility(view)
+        return view
+    }
+
+    override fun onResume() {
+        super.onResume()
+        val width = ConstraintLayout.LayoutParams.MATCH_PARENT
+        val height = ConstraintLayout.LayoutParams.WRAP_CONTENT
+        dialog!!.window!!.setLayout(width, height)
+    }
```

Comment:
This is not set by default when you resume?

Reply:
Nop, the fragments looks better with these constraints.